lib/ansible/galaxy/collection/__init__.py

```diff
@@ -246,6 +263,26 @@ def verify_local_collection(
     return result
 
 
+def verify_file_signature(b_manifest_file, b_detached_signature, keyring):
+    gpg_result, rc = run_gpg_verify(b_manifest_file, b_detached_signature, keyring, display)
+
+    errors = None
+    if gpg_result:
+        errors = parse_gpg_errors(gpg_result)
+
+    if errors:
```

Comment:
Wonder if this could be something like
```suggestion
    if (gpg_result and errors := parse_gpg_errors(gpg_result)):
```

Reply:
Perfect opportunity to use the walrus operator :100: